src/foam/nanos/crunch/lite/CapableDAO.js

```diff
@@ -108,7 +108,7 @@ foam.CLASS({
         List<CapabilityJunctionPayload> capablePayloads = new ArrayList<CapabilityJunctionPayload>(Arrays.asList(toPutCapablePayloadArray));
 
         for ( CapabilityJunctionPayload currentPayload : capablePayloads ){
-          toUpdateCapablePayloadDAO.inX(x).put(currentPayload);
+          toUpdateCapablePayloadDAO.inX(x).put((CapabilityJunctionPayload) currentPayload.fclone());
```

Comment:
Please add a comment to explain why this is required.

Reply:
this line is throwing an exception on the server where object is frozen 
```2023-01-23T09:57:56.001-0500,qtp2042884127-160,DEBUG,[Service],analyticEventDAO,spid,intuit,user,34223,session,3ca9363c,returning skeleton exception,java.lang.UnsupportedOperationException: Object is frozen.
    at foam.core.FObject.assertNotFrozen(FObject.java:620)
    at net.nanopay.gateway.GatewayPaymentRequest.setCapablePayloads(GatewayPaymentRequest.java:2040)
    at foam.nanos.crunch.lite.CapableAdapterDAO.put_(CapableAdapterDAO.java:254)
    at foam.nanos.ruler.RulerDAO.put_(RulerDAO.java:2051)
    at foam.dao.ProxyDAO.put_(ProxyDAO.java:466)
    at foam.dao.ProxyDAO.put_(ProxyDAO.java:466)
    at foam.dao.ProxyDAO.put_(ProxyDAO.java:466)
    at foam.dao.AbstractDAO.put(AbstractDAO.java:674)
    at foam.nanos.crunch.lite.CapableDAO.put_(CapableDAO.java:540) <<< here
    at foam.dao.ProxyDAO.put_(ProxyDAO.java:466)
    at foam.nanos.auth.AuthorizationDAO.put_(AuthorizationDAO.java:283)
    at foam.dao.ProxyDAO.put_(ProxyDAO.java:466)
    at foam.nanos.auth.PermissionedPropertyDAO.put_(PermissionedPropertyDAO.java:121)
    at foam.dao.ProxyDAO.put_(ProxyDAO.java:466)
    at foam.dao.ProxyDAO.put_(ProxyDAO.java:466)
    at foam.dao.ProxyDAO.put_(ProxyDAO.java:466)
    at foam.dao.DAOSkeleton.send(DAOSkeleton.java:175)
    at foam.dao.SessionDAOSkeleton.send(SessionDAOSkeleton.java:53)
    at foam.box.SessionServerBox.send(SessionServerBox.java:195)
    at foam.box.SessionServerBox.send(SessionServerBox.java:54)
    at foam.nanos.http.ServiceWebAgent.execute(ServiceWebAgent.java:137)
    at foam.nanos.http.NanoRouter.service(NanoRouter.java:128)

Reply:
Still seeing the exception at the end of gateway token, is it throwing in CapableAdapterDAO not in CapableDAO? `foam.nanos.crunch.lite.CapableAdapterDAO.put_(CapableAdapterDAO.java:254)`.